src/libs/ReportUtils.ts

```diff
@@ -816,10 +818,17 @@ function isConciergeChatReport(report: OnyxEntry<Report>): boolean {
 /**
  * Returns true if report is still being processed
  */
-function isProcessingReport(report: OnyxEntry<Report>): boolean {
+function isProcessingReport(report: OnyxEntry<Report> | EmptyObject): boolean {
     return report?.stateNum === CONST.REPORT.STATE_NUM.PROCESSING && report?.statusNum === CONST.REPORT.STATUS.SUBMITTED;
 }
 
+/**
+ * Returns true if report is still open
+ */
+function isOpenReport(report: OnyxEntry<Report> | EmptyObject): boolean {
+    return report?.stateNum === CONST.REPORT.STATE_NUM.OPEN && report?.statusNum === CONST.REPORT.STATUS.OPEN;
+}
+
```

Comment:
We already have a method to check for drafts, also we use drafts in the App so I think thats a better name 

https://github.com/Expensify/App/blob/daf8d0de590ff3a89d497e9f9b16532f01c340a7/src/libs/ReportUtils.ts#L606-L611

IOU reports cannot be open so the check for the report type is fine there

Reply:
are you suggesting I remove `isExpenseReport` from this function?

Reply:
No no, no need to remove it